### PR TITLE
{bio} [goolf-1.4.10] VCFtools-0.1.14  (REVIEW)  

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    (perl , perlver),
+]   
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["vcftools", "fill-fs", "vcf-compare", "vcf-contrast", 
+                                     "vcf-fix-ploidy", "vcf-merge"]],
+    'dirs': [],
+}
+
+modextrapaths = { 'PERL5LIB': 'lib/perl5/site_perl/%s/' % (perlver) }
```
Comment (boegel) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb]: existing easyconfig for VCFtools 0.1.14 has this instead:

```python
modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl'}
```

One of these is wrong?

Also, style-wise this should be:

```python
modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl/%s' % perlver}
```

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description = """A set of tools written in Perl and C++ for working with VCF files."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/vcftools/vcftools/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+perl = 'Perl'
+perlver = '5.16.3'
+versionsuffix = '-%s-%s' % (perl, perlver)
```
Comment (boegel) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb]: use `%(perlver)s`?

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+dependencies = [
+    (perl , perlver),
```
Comment (boegel) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb]: style nitpicking: no space before `,`, only agter

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/baa5db3b354c1f635d1e for a full test report.

**Comment by boegel** [comment]
@pescobar ping on remarks?

**Comment by pescobar** [comment]
@boegel I added both `lib/perl5/site_perl/%(perlver)s/` and `lib/perl5/site_perl/` to `$PERL5LIB` to be on the safe side...

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb]: boegel the contents of `lib/perl5/site_perl/%(perlver)s/` are these: `FaSlice.pm  Vcf.pm  VcfStats.pm` 

so I think the right one is `lib/perl5/site_perl/%(perlver)s/`

**Comment by pescobar** [comment]
I am doing some tests and both options work for $PERL5LIB

```
[escobar@login18 ~]$ ls $EBROOTVCFTOOLS/lib/perl5/site_perl/5.22.2/
FaSlice.pm  Vcf.pm  VcfStats.pm

[escobar@login18 ~]$ grep FaSlice `which vcf-compare`
use FaSlice;

[escobar@login18 ~]$ echo $PERL5LIB 
/scicore/home/scicore/escobar/.local/easybuild/software/VCFtools/0.1.14-goolf-1.7.20-Perl-5.22.2/lib/perl5/site_perl/5.22.2/

[escobar@login18 ~]$ vcf-compare -h
About: Compare bgzipped and tabix indexed VCF files. (E.g. bgzip file.vcf; tabix -p vcf file.vcf.gz)
Usage: vcf-compare [OPTIONS] file1.vcf file2.vcf ...
       vcf-compare -p plots chr1.cmp chr2.cmp ...

[escobar@login18 ~]$ export PERL5LIB=$EBROOTVCFTOOLS/lib/perl5/site_perl/

[escobar@login18 ~]$ echo $PERL5LIB 
/scicore/home/scicore/escobar/.local/easybuild/software/VCFtools/0.1.14-goolf-1.7.20-Perl-5.22.2/lib/perl5/site_perl/

[escobar@login18 ~]$ vcf-compare --help
About: Compare bgzipped and tabix indexed VCF files. (E.g. bgzip file.vcf; tabix -p vcf file.vcf.gz)
Usage: vcf-compare [OPTIONS] file1.vcf file2.vcf ...
       vcf-compare -p plots chr1.cmp chr2.cmp ...

```

So I go for `modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl'}`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/16d1aaad421bf58d5b57fe14709286ad for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**